### ISSUE TITLE
Unrelated bug while editing an expression via statement editor in data mapper

### ISSUE THREAD

**Issue by kaviththiranga** [wso2/ballerina-plugin-vscode#263]
**Description:**
<img width="1413" alt="image" src="https://user-images.githubusercontent.com/1505855/200781356-0f1c6056-29c7-4cd4-a3f8-fed0060b9412.png">

```ballerina


function transform(Person person) returns Student => {

    courses: from var courseItem in person.course
        select {
            title: courseItem.name,
            credits: courseItem.credits
        },
    age: person.person.age.toBalString(),
    fullName: person.person.firstName + " " 
                + person.person.lastName,
    id: person.person.id,
    totalCredits: 0

    
};

type Person record {
    record {
        int id;
        string firstName;
        string lastName;
        int age;
    } person;
    record {
        string id;
        string name;
        int credits;
    }[] course;
};

type Student record {
    int id;
    string fullName;
    string age;
    record {
        string title;
        int credits;
    }[] courses;
    int totalCredits;
};


```

**Steps to reproduce:**

**Affected Versions:**

```
Ballerina 2201.2.3 (Swan Lake Update 2)
Language specification 2022R3
Update Tool 1.3.10

Extension: v3.3.3
```

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

**Comment by madushajg** [comment]
Seems like a duplicate of https://github.com/wso2/ballerina-plugin-vscode/issues/239.
The fix will be available in extension v3.3.4

**Comment by kaumini** [comment]
Fixed with https://github.com/wso2-enterprise/ballerina-low-code-editor/pull/1493
![Screenshot from 2023-02-01 16-00-20](https://user-images.githubusercontent.com/36252572/216018735-b8c40f0c-df70-41c0-afbd-b425f7a0d5e2.png)
